integration-tests/nojs-test.js

```diff
@@ -0,0 +1,23 @@
+const { expect } = require('chai');
+const visit = require('./helpers/visit');
+
+describe('the main flow without javascript', function() {
+  it('works', async function() {
+    await visit('/', { disableJavascript: false }, async (page) => {
```

Comment:
Should this not be `disableJavascript: true`?

Reply:
Oh yeah, thanks. I must have missed it when I was experimenting with configs.